sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -6945,7 +6945,7 @@ private void WithUsings(NamedDecl namedDecl)
     {
         Debug.Assert(_outputBuilder is not null);
 
-        if (TryGetRemappedValue(namedDecl, _config.WithUsings, out var usings))
+        if (TryGetRemappedValue(namedDecl, _config.WithUsings, out var usings, matchStar: true))
```

Comment:
Can you add a test covering this working?

Reply:
Done - there didn't seem to be a good place to test how options affect the output, so I created a new test class.